__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+let NEWS_ID = '';
```

Comment:
There is no need to uppercase this variable, it is not a constant, it is going to be changed anyway and it exists only for the purpose of the test, so no need to uppercase it. The same thing for the next one

Reply:
fixed